test/main.js

```diff
@@ -16,6 +16,7 @@ if (!process.env.BROWSERSTACK_ACCESS_KEY) {
 // Input capabilities
 const bsCapabilitiesDefault = {
   'acceptSslCerts' : 'true',
+  'forceLocal': 'true',
```

Comment:
What does this do?

Reply:
From BS: https://www.browserstack.com/local-testing/automate#test-websites-hosted-private-internal-servers

`You can securely test websites hosted on private or internal servers with Local Testing using --force-local capability. This option resolves all requests (on our remote browsers and devices) through your local machine.`